wradlib/util.py

```diff
@@ -987,3 +989,233 @@ def has_import(module):
 
 if __name__ == "__main__":
     print("wradlib: Calling module <util> as main...")
+
+
+def vertical_interpolation(vol, elevs=None, method="nearest"):
+    """
+    Vertically interpolate volume data
+
```

Comment:
Please add proper `Parameters` and `Keyword Arguments` in the docstring.

Maybe change function name to `vertical_interpolate_volume`.